### PR TITLE
Refactor CSS by putting px values into values

### LINKED ISSUE #299: Refactor CSS by putting px values into a _values.scss
We should organize our px values in the CSS. Create a `_values.scss` file, similar to `_colors.scss`, and replace all of the px values with variables.

### PR DESCRIPTION
**Pull request by zachariast**
change all the font sizes px with values

fix  #299

### CODE REVIEW

File: app/assets/stylesheets/base/_values.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// This file contains the base fonts-sizes for the application.
+
+$body: 20px;
```
Comment (Wowu) [inline app/assets/stylesheets/base/_values.scss]: I think `$body` variable name is too general and not so obvious. This file can have more variables in the future. What about `body_font` or something like that?

Comment (zachariast) [inline app/assets/stylesheets/base/_values.scss]: @Wowu Yes of course.

**Comment by zachariast** [comment]
@Wowu for some reason travis build was failed.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -37,28 +37,28 @@
 
 body, html {
   margin: 0 auto;
-  font: 20px $font-family;
+  font: $body_font_size $font-family;
```
Comment (julianguyen) [inline app/assets/stylesheets/application.scss]: Just to be consistent, let's use kabobcase e.g. $font-family

Comment (zachariast) [inline app/assets/stylesheets/application.scss]: @julianguyen yes, kabobcase....done. :)

Comment (zachariast) [inline app/assets/stylesheets/application.scss]: @julianguyen should i merge my commits?

**Comment by Wowu** [comment]
There are still px font sizes in other scss files in `/app/assets/stylesheets/` folder

**Comment by zachariast** [comment]
@Wowu i noticed now. In application folder. I'll work on it.

**Comment by Wowu** [comment]
LGTM :smile:

**Pull request merged**